pkgs/development/python-modules/python-etcd/default.nix

```diff
@@ -34,8 +36,11 @@ buildPythonPackage rec {
     sed -i '19s/dns/"dnspython"/' setup.py
   '';
 
-  # Some issues with etcd not in path even though most tests passed
-  doCheck = false;
+  nativeCheckInputs = [ pytestCheckHook ];
+
+  preCheck = ''
+    export PATH="${etcd_3_4}/bin:$PATH"
+  '';
```

Comment:
Couldn’t this just be in `nativeCheckInputs`? I take it the tests don’t pass with our default etcd 3.5?

Reply:
Cant replace with `etcd` for 2 reasons:

- `etcd` has a name collision with `python3Packages.etcd` so it wod require importing `pkgs` (`pkgs.etcd`)
- tests dont pass with `pkgs.etcd`, only with the `3.4` version